### PR TITLE
docs: add swift package manager info to ios reference

### PR DESCRIPTION
**Pull request by cre8ivejp**

### CODE REVIEW

File: docs/sdk/client-side/ios/index.md [hunk 1]
```diff
@@ -33,6 +35,27 @@ target 'YOUR_TARGET_NAME' do
...
 
+</TabItem>
+<TabItem value="spm" label="Swift Package Manager">
+
+You can install the SDK as a dependency in your **Package.swift** file or through Xcode.
+
+```swift showLineNumbers
+// Package.swift
+dependencies: [
+  .package(url: "https://github.com/bucketeer-io/ios-client-sdk.git", .exact("LATEST_VERSION")),
```
Comment (s2mr) [inline docs/sdk/client-side/ios/index.md:46]: `.exact` is deprecated.
`exact:` is better than it.

```suggestion
  .package(url: "https://github.com/bucketeer-io/ios-client-sdk.git", exact: "LATEST_VERSION"),
```

File: docs/sdk/client-side/ios/index.md [hunk 2]
```diff
@@ -33,6 +35,27 @@ target 'YOUR_TARGET_NAME' do
...
+],
+targets: [
+  .target(
+    name: "YOUR_TARGET_NAME",
+    dependencies: ["Bucketeer"]
```
Comment (s2mr) [inline docs/sdk/client-side/ios/index.md:51]: Its usage is deprecated.
For this, please provide a specific type and package name.

```suggestion
    dependencies: [.product(name: "Bucketeer", package: "ios-client-sdk")]),
```

**Review by s2mr — DISMISSED** [review]
Commented.
It is a minor point and I will approve it.

**Comment by cre8ivejp** [comment]
@s2mr, I have fixed it! Thank you!

**Pull request merged**